src/mysky.ts

```diff
@@ -219,6 +262,40 @@ export class MySky {
     return publicKey;
   }
 
+  /**
+   * verifySignedMessage verifies the given message and returns the original
+   * message without signature if verification succeeded
+   *
+   * @param signedMsg - the signed message that needs to be verified
+   * @param publicKey - the public key to verify against
+   * @returns original message, or null in case verification failed
+   */
+  async verifySignedMessage(signedMsg: Uint8Array, publicKey: Uint8Array | string): Promise<Uint8Array | null> {
```

Comment:
Why not pass in the original message + separate signature? It's what we do elsewhere in the SDK and MySky so would be good for consistency, but apologies if I missed something.